registrations/serializers.py

```diff
@@ -158,3 +163,37 @@ def save(self):
                 email
             )
         )
+
+class UserExternalTokenSerializer(serializers.ModelSerializer):
+    user_details = UserNameSerializer(source="user", read_only=True)
+    class Meta:
+        model = UserExternalToken
+        fields = 'id', 'user_details', 'token', 'expire_timestamp'
+
+    def validate_expire_timestamp(self, data):
+        if data < timezone.now():
+            raise serializers.ValidationError('Expire timestamp must be in the future.')
+        return data
```

Comment:
Let's add a max limit as well

```suggestion
    def validate_expire_timestamp(self, date):
        if date < timezone.now():
            raise serializers.ValidationError('Expire timestamp must be in the future.')
        return date
```